frontend/packages/noobaa-storage-plugin/src/queries.ts

```diff
@@ -2,21 +2,26 @@ import { ProjectModel } from '@console/internal/models';
 import { PROJECTS, BUCKET_CLASS } from './constants';
```

Comment:
can we call it `dashboard-queries.ts` to be more clear?

Reply:
Its possible that other noobaa components would use same queries in future, so we can keep it as it is for now.